### PR TITLE
Updated callers to Holmes

### PR DESCRIPTION
**Pull request by andrewpatto**

### CODE REVIEW

**Comment by andrewpatto** [comment]
diff-extract has been removed from new version so just cleaning up that code as well

**Review by victorskl — APPROVED** [review]
LGTM otherwise!

(Side note: if ctTSO is not `hg38.rna`, I'd expect we shall have another PR for its patch. That's probably by Alexis, I reckon?)

EDIT: It will be #509. Ditto convo below.

File: data_processors/pipeline/domain/somalier.py [hunk 1]
```diff
@@ -148,19 +143,20 @@ def poll(self):
         self.execution_result = execution_dict
         return self
 
-    def extract(self, instance_name, gds_path):
+    def extract(self, instance_name, gds_path, reference = "hg38.rna"):
         """payload bound to holmes extract interface
         https://github.com/umccr/holmes#extract
+        Reference which defaults to hg38 has been added
+        ctTSO will need to call extract with alternative reference
```
[the remark this answers is not in the document]
Comment (alexiswl) [reply data_processors/pipeline/domain/somalier.py:150]: Should just be `hg19.rna`?  Right @patto?

[the remark this answers is not in the document]
Comment (victorskl) [reply data_processors/pipeline/domain/somalier.py:150]: Right! I can make that part of changes for ctTSO flow. Just need to know/confirm the reference name for ctTSO...

[the remark this answers is not in the document]
Comment (alexiswl) [reply data_processors/pipeline/domain/somalier.py:150]: Yup according to this it's hg19.rna

https://github.com/umccr/holmes/blob/b25738a045c82e7cf8569ca58d0edbe0239bdb1f/application/fingerprint-docker-image/lib/extract.ts#L160-L165

[the remark this answers is not in the document]
Comment (andrewpatto) [reply data_processors/pipeline/domain/somalier.py:150]: yes the ctTSO is ready from the holmes end to be specified as "hg19.rna" - but re-enabling the ctTSO felt like a bigger patch so was going to put that off for the main February release. But we can do now if you want?

[the remark this answers is not in the document]
Comment (victorskl) [reply data_processors/pipeline/domain/somalier.py:150]: From @andrewpatto 

> yes the ctTSO is ready from the holmes end to be specified as "hg19.rna" - but re-enabling the ctTSO felt like a bigger patch so was going to put that off for the main February release. But we can do now if you want?

[the remark this answers is not in the document]
Comment (victorskl) [reply data_processors/pipeline/domain/somalier.py:150]: Ah right! That ctTSO is for this ticket -- https://github.com/umccr/data-portal-apis/issues/509

Sorry, I was confused, thought we already have TSO BAM finger printing...

Yup, yup. It is for better go with that tickie #509 in next release then. Not for this patch, yes!

I will merge this now and, start testing tick, next...

[the remark this answers is not in the document]
Comment (victorskl) [reply data_processors/pipeline/domain/somalier.py:150]: > Do we want to put the code for triggering the ctTSOs into this PR?

Nop. Not yet. Sorry, me was confused, thought we have that which is not, it is still disabled.

I will work on reenabling ctTSO in next release...

**Pull request merged**